src/screens/OrgList/OrganizationModal.spec.tsx

```diff
@@ -16,8 +16,14 @@ import OrganizationModal from './OrganizationModal';
 import i18nForTest from 'utils/i18nForTest';
 import userEvent from '@testing-library/user-event';
 
-vi.mock('utils/convertToBase64', () => ({
-  default: vi.fn(() => Promise.resolve('mockBase64String')),
+const mockUploadFileToMinio = vi
+  .fn()
+  .mockResolvedValue({ fileUrl: 'https://minio-test.com/test-image.jpg' });
```

Comment:
1. The develop-postgres branch of the API no longer uses `fileurl` therefore your mock is incorrect. Using `fileurl` in the app will cause it to malfunction.
2. You need to use `objectName` instead 
3. You also cannot use `objectName` in the preview. This is impossible. You have to use the  browser's URL object method to preview the images and the MinioUpload.ts helper function for uploading images.